.github/workflows/workflows/codeql.yml

```
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * 0'
```

qq: this action will run on every single push, so what extra benefit one would get by running it daily?

---

The set of security queries we use gets better all the time. If you don't push very often (not the case on sourcegraph/sourcegraph!) then running once a week will sometimes surface new alerts due to query improvements

---

Gotcha, make sense! Weekly makes the most sense for sourcegraph/sourcegraph I think (ideally, Monday 00:00 but cc @nicksnyder here).

So if a new alert is surfaced, in which form that we will be notified (issue, PR, or)?

---

It's be in the security tab on the repo. We don't currently send notifications for them but will add that in future.

The below will do weekly on Monday at 00:00 UTC (and the previous setup was for weekly - Justin misspoke in the PR text).

```suggestion
    - cron: '0 0 * * 1'
```